app/assets/stylesheets/_cookie-banner.scss

```diff
@@ -0,0 +1,56 @@
+.cookies-banner {
+  background-color: $lightgrey-10;
+  position: fixed;
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;  
+  border-radius: 4px;
+  margin: 0 auto;
+  padding: 10px;
+  left: 20px;
+  right: 20px;
+  max-width: fit-content;
+  bottom: -400px;
+  box-shadow: $lightgrey-16 0px 0px 12px 0px;
+  z-index: 1000;
+  visibility: hidden;
+  transition: all 0.8s ease-in-out;  
```

Comment:
What do you think about changing it to say that only the properties that change will transition? This would be an improvement in the semantics of this code + a small performance gain.